contrib/macdeploy/README.md

```diff
@@ -15,8 +15,10 @@ When complete, it will have produced `Bitcoin-Core.dmg`.
 Our current macOS SDK
 (`Xcode-12.1-12A7403-extracted-SDK-with-libcxx-headers.tar.gz`) can be
 extracted from
-[Xcode_12.1.xip](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip).
+[Xcode_12.1.xip](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip)
+(alternatively, after login to your account go to "Downloads", then "More" and look for [`Xcode_12.1`](https://download.developer.apple.com/Developer_Tools/Xcode_12.1/Xcode_12.1.xip)).
```

Comment:
Would there be any point in linking to this one? 

https://bitcoincore.org/depends-sources/sdks/Xcode-12.1-12A7403-extracted-SDK-with-libcxx-headers.tar.gz

Reply:
you're not allowed to redistribute the SDK, that's a fake link 🥸